[Doc] Update field.md (#6182)
taichi-dev/taichi: docs/lang/articles/basic/field.md

          f_1d.shape = (9,)
  ```

>   There is no essential difference between a 0D field and a 1D field of length 1, except that you must use `None` as the index to access a 0D field, whereas you must use `0` as the index to access a 1D field of length 1:

Reviewer: ```suggestion
There is no essential difference between a 0D field and a 1D field of length 1, except the indexing rules. You *must* use `None` as the index to access a 0D field and use `0` as the index to access a 1D field of length 1:
```

>     There is no essential difference between a 0D field and a 1D field of length 1, except the indexing rules. You *must* use `None` as the index to access a 0D field and use `0` as the index to access a 1D field of length 1:

Reviewer 2: ```suggestion
    There is little difference between a 0D field and a 1D field of length 1 except for the indexing rules. You *must* use `None` as the index to access a 0D field and `0` as the index to access a 1D field of length 1:
```

  ```python

>   Basially f1 and f2 are interchangeable

Reviewer: Missing a #?

  f1 = ti.field(int, shape=())
  f2 = ti.field(int, shape=1)

  f1[None] = 1  # Use None to access a 0D field
  f2[0] = 1  # Use 0 to access a 1D field of length 1
  ```

> - To declare a 2D scalar field, set its two dimensions respectively, i.e., the numbers of rows and columns. For example, the following code snippet defines a 2D scalar field of shape (3, 6) (of 3 rows and 6 columns):

Reviewer: ```suggestion
- To declare a 2D scalar field, set its two dimensions respectively, i.e., the numbers of rows and columns. For example, the following code snippet defines a 2D scalar field of shape `(3, 6)` (i.e., three rows and six columns):
```

Reviewer 2: We do not use i.e. or e.g. in technical documents. i.e. can be omitted in this case.

Reviewer 2: Please use this version:

To declare a 2D scalar field, set its two dimensions (numbers of rows and columns) respectively. For example, the following code snippet defines a 2D scalar field in the shape `(3, 6)` (three rows and six columns):

Reviewer 2: I think this version is better: 

When declaring a 2D scalar field, you need to set its two dimensions (numbers of rows and columns) respectively. For example, the following code snippet defines a 2D scalar field in the shape (3, 6) (three rows and six columns):

----
It is incorrect to say that the purpose of setting two dimensions of a field is to declare it.
